app/samples/sample-browsemap-markers/src/main/java/com/sygic/samples/BrowseMapMarkersActivity.kt

```diff
@@ -14,29 +15,28 @@ class BrowseMapMarkersActivity : CommonSampleActivity() {
 
         setContentView(R.layout.activity_browsemap_markers)
 
-        val markerFromBuilder = MapMarker.Builder()
-            .coordinates(48.130550, 17.173795)
-            .title("Marker created by Builder")
+        val markerFromBuilder = MapMarker.from(48.130550, 17.173795)
```

Comment:
stale ked sa na to pozeram mam pocit ze lepsie ako `z` (pozicie) by bolo `na` (pozicii), kedze to je takmer vsade tak pouzite. Aj ked viem ze su tam aj z `ProxyPoi` a `z UiUobject` atd :/

Reply:
A aj tie ostatne nazvy builder metod su take kostrbate. napr `.payload()` by som cakal skor nieco ako `.withPayload()`. Je to viac pismen, ale je to take citatelnejsie

Reply:
přejmenuji